task_storing_groceries/scripts/demo.py

```diff
@@ -0,0 +1,353 @@
+#! /usr/bin/env python
```

Comment:
Is this script working or for development use? From first glance it looks like it shouldn't be tracked due to the commented out chunks.